docs/conf.py

```diff
@@ -60,7 +60,7 @@ def setup(app):
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
```

Comment:
suppresses `sphinx-build` warning (we currently don't have any plan to put `_static` by ourselves).